lib/to_duration/converter.rb

```diff
@@ -42,6 +42,7 @@ def call
     }.freeze
 
     def to_units
+      return ["< 1 #{I18n.t('to_duration.second', count: 1)}"] if @value < 1
```

Comment:
`return [I18n.t('to_duration.less_than_one_second')] if @value < 1` looks better.

Also, do not forget to edit `test/test_data/locales/en.yml` to pass the test, e.g:

```yml
en:
  to_duration:
    less_than_one_second: 'Less than 1 second'
```